uisdk/src/androidTest/java/com/karhoo/uisdk/common/ResponseUtils.kt

```diff
@@ -1,17 +1,21 @@
 package com.karhoo.uisdk.common
 
+import android.util.Log
 import androidx.test.platform.app.InstrumentationRegistry
 import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder
 import com.github.tomakehurst.wiremock.client.WireMock
+import java.io.BufferedReader
 
-class ResponseUtils(val httpCode: Int, val fileName: String, val useJson: Boolean = false, val delayInMillis: Int = 0) {
+class ResponseUtils(val httpCode: Int, val response: String, val useJson: Boolean = false, val
+delayInMillis: Int = 0) {
 
     fun createResponse(): ResponseDefinitionBuilder {
         val json = if (!useJson) {
             getJsonFromFile()
         } else {
-            fileName
+            response
         }
+        Log.d("Adyen", json)
```

Comment:
Is this log accurate? Could it be Braintree or any other component?

Reply:
Deleted